dev/Deployment/DeploymentManager.cpp

```diff
@@ -8,8 +8,12 @@
 #include <TerminalVelocityFeatures-DeploymentAPI.h>
 #include <Microsoft.Windows.ApplicationModel.WindowsAppRuntime.DeploymentManager.g.cpp>
 
+#include <WinBase.h>
```

Comment:
Well that's weird. Why? Isn't that already included by stuff in pch.h?

winbase.h is included by windows.h (or unknwn.h)

Or is this a same named but different file from something else? C++/WinRT ?